actions/ui-check/tests/e2e/specs/page/index.test.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * External dependencies
+ */
+import { createURL } from '@wordpress/e2e-test-utils';
+import { getPageError } from '@wordpress/e2e-test-utils';
+const core = require( '@actions/core' );
+
+jest.setTimeout( 1000000 );
```

Comment:
This is already in the main branch, I introduced it earlier. This can be removed.

Reply:
Here:

https://github.com/StevenDufresne/wp-theme-validation-action-experiment/blob/a9a1b808e494a8baa11f3aabae718cc80b2f2591/actions/ui-check/tests/e2e/jest.config.js#L3